rollbar/__init__.py

```diff
@@ -1606,3 +1632,34 @@ def _wsgi_extract_user_ip(environ):
     if real_ip:
         return real_ip
     return environ['REMOTE_ADDR']
+
+
+scopes = []
+
+
+class _BigBrother(object):
```

Comment:
I am not sure this is a good name for the context manager class. What it tries to represent?

Reply:
Ah. This was a reference to George Orwell's _1984_. I thought of this because we decided on using the method name `watch` and it reminded me of the slogan "Big Brother is watching you".

I thought it was funny 😅

Reply:
Ah. So when we decided on the method name `watch`, it reminded me of the slogan, "Big Brother is watching you" from George Orwell's _1984_, thus `_BigBrother`.

I thought it'd be fun 😅

Reply:
ahm...well...it might be fun but doesn't represent what it does, maybe we can change it

Reply:
Haha. I can change it.